src/main/webapp/src/app/admin-page/admin-page.component.css

```diff
@@ -0,0 +1,24 @@
+.card-padding {
+  padding: 20px;
+}
+
+.button-style {
+  display: block;
+  padding: 0.5em 3em;
+  text-decoration: none;
+  border: none;
+  box-sizing: border-box;
+  box-shadow: inset 0 -0.6em 0 -0.35em #003353;
+  background-color: #005c80;
+  color: white;
+}
+
+.button-style:hover {
+  box-shadow: inset 0 -0.6em 0 -0.35em whitesmoke;
+  background-color: white;
+  color: #005c80;
+}
```

Comment:
Optional: Since we are using the buttons in several pages, it might be a good idea to move the button styling inside the common folder instead

Reply:
I actually don't think I have anything tagged `button-style` -- I use another style from `common`. Will remove this style.